apps/total-typescript/src/components/exercise-overlay.tsx

```diff
@@ -114,9 +114,9 @@ const Actions = () => {
 }
 
 const ExerciseOverlay = () => {
-  const {lesson, module} = useMuxPlayer()
+  const {lesson, module, lessonMedia} = useMuxPlayer()
```

Comment:
the introduction of `lessonMedia` seperates playback ids and stackblitz from the static data that is loaded at buildtime